source/connect/in-use-encryption.txt

```diff
@@ -0,0 +1,233 @@
+            manage your keys.
+
+            Specify the following fields:
+
+            .. list-table::
+               :header-rows: 1 
+               :widths: 50 50
+
+               * - Field
+                 - Description 
+
+               * - Service Account Email
+                 - The service account email to authenticate.
+
+               * - Private Key
+                 - A base64-encoded private key.
+
+               * - Endpoint
+                 - (Optional) A host with an optional port.
+
+               * - Certificate Authority
+                 - (Optional) One or more certificate files from trusted Certificate 
+                   Authorities to validate the certificate provided by the deployment.
+               
+               * - Client Certificate and Key
+                 - (Optional) Specifies the location of a local .pem file that contains 
+                   either the client's TLS/SSL X.509 certificate or the client's TLS/SSL 
+                   certificate and key.
+                
+               * - Client Key Password
+                 - (Optional) If the *Client Private Key* is protected with a password, 
+                   you must provide the password.
+
+            .. _azure-compass-encryption:
+
+            Azure
+            ~~~~~
+
+            You can use :manual:`Azure Key Vault </core/security-client-side-encryption-key-management/#std-label-field-level-encryption-azure-keyvault/>` to manage your 
+            keys.
+
+            Specify the following fields:
+
+            .. list-table::
+               :header-rows: 1 
+               :widths: 50 50
+
+               * - Field
+                 - Description 
+
+               * - Tenant Id 
+                 - Identifies the organization for the account.
+
+               * - Client Id 
+                 - Authenticates a registered application.
+
+               * - Client Secret
+                 - The client secret to authenticate a registered application.
+
+               * - Identity Platform Endpoint
+                 - A host with an optional port. 
+
+               * - Certificate Authority
+                 - (Optional) One or more certificate files from trusted Certificate 
+                   Authorities to validate the certificate provided by the deployment.
+               
+               * - Client Certificate and Key
+                 - (Optional) Specifies the location of a local .pem file that contains 
+                   either the client's TLS/SSL X.509 certificate or the client's TLS/SSL 
+                   certificate and key.
+                
+               * - Client Key Password
+                 - (Optional) If the *Client Private Key* is protected with a password, 
+                   you must provide the password.  
+
+
+            .. _kmip-compass-encryption:
+
+            KMIP
+            ~~~~
+
+            You can use :manual:`KMIP </core/csfle/tutorials/kmip/kmip-automatic/>` 
```

Comment:
This link is 404ing for me, perhaps you could use:

```
:ref:`KMIP <csfle-tutorial-automatic-kmip>`
```
Instead?

Reply:
Done.